code/modules/exploration_crew/exploration_explosives.dm

```diff
@@ -0,0 +1,126 @@
+/obj/item/grenade/exploration
+	name = "breaching charges"
+	desc = "Used by exploration crews to breach and demolish unknown places. Requires a nearby detonator to be triggered."
+	icon_state = "plastic-explosive0"
+	item_state = "plastic-explosive"
+	lefthand_file = 'icons/mob/inhands/weapons/bombs_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/bombs_righthand.dmi'
+	item_flags = NOBLUDGEON
+	flags_1 = NONE
+	w_class = WEIGHT_CLASS_SMALL
+	var/atom/target = null
+	var/mutable_appearance/plastic_overlay
+	var/boom_sizes = list(0, 0, 3)
```

Comment:
```suggestion
	var/static/boom_sizes = list(0, 0, 3)
```

Reply:
boom_sizes could be changed by subtypes and having it static wouldn't allow for the subtypes to change it. Currently there aren't subtypes so I can see why this wouldn't want to be changed, but the better thing to do would be to just input the raw values and remove the boom_sizes var entirely instead of having a pointless static var.

Reply:
yeah remove the var

Reply:
nah but ill make it not a list because its cleaner